package.json

```diff
@@ -15,16 +15,20 @@
     "@fortawesome/free-brands-svg-icons": "^5.0.0",
     "@fortawesome/free-solid-svg-icons": "^5.0.0",
     "@fortawesome/react-fontawesome": "^0.1.0",
+    "@rehooks/window-size": "^1.0.2",
     "@storybook/addon-actions": "^4.0.0",
     "emotion": "^9.0.0",
     "grommet": "^2.0.0",
-    "moment": "^2.0.0",
-    "polished": "^2.0.0",
+    "moment": "^2.23.0",
+    "polished": "^2.3.3",
+    "react-collapsible": "^2.6.0",
+    "react-inlinesvg": "^0.8.4",
     "react-styled-flexboxgrid": "^3.0.0",
-    "react-textarea-autosize": "^7.0.0",
+    "react-textarea-autosize": "^7.1.0",
+    "react-use-scroll-position": "^1.0.0",
     "reactstrap": "^6.0.0",
-    "styled-components": "^4.0.0",
-    "styled-normalize": "^8.0.0"
+    "styled-components": "^4.1.3",
```

Comment:
no need to do such updates. Just leave it at `^4.0.0` and do `yarn upgrade` if you want a newer version

Reply:
alright, I'm always a bit afraid that everything will break if I upgrade everything ;)